resources/assets/components/Quantity/index.js

```diff
@@ -20,7 +20,11 @@ class Quantity extends React.Component {
 
 Quantity.propTypes = {
   noun: PropTypes.string,
-  quantity: PropTypes.string,
+  // @TODO - figure out why this comes in two different ways.
```

Comment:
Oy! I wonder if this is another case where we sometimes send it through the transformer & sometimes don't.